src/test/scala/com/campudus/tableaux/LinkTest.scala

```diff
@@ -209,6 +199,173 @@ class LinkTest extends TableauxTestBase {
     }
   }
 
+  private def setupTablesWithEmptyLinks(): Future[Number] = {
+    val linkColumn = Json.obj(
+      "columns" -> Json.arr(
+        Json.obj(
+          "name" -> "Test Link 1",
+          "kind" -> "link",
+          "fromColumn" -> 1,
+          "toTable" -> 2,
+          "toColumn" -> 1
+        )
+      )
+    )
+
+    def addRow(tableId: Long, values: JsonObject): Future[Number] = for {
+      res <- sendRequestWithJson("POST", values, s"/tables/$tableId/rows")
+      table1RowId1 <- Future.apply(res.getArray("rows").get[JsonObject](0).getNumber("id"))
+    } yield table1RowId1
+
+    def valuesRow(c: String) = Json.obj(
+      "columns" -> Json.arr(Json.obj("id" -> 1), Json.obj("id" -> 2)),
+      "rows" -> Json.arr(Json.obj("values" -> Json.arr(c, 2)))
+    )
+
+    for {
+    // setup two tables
+      tables <- setupTables()
+
+      // create link column
+      res <- sendRequestWithJson("POST", linkColumn, "/tables/1/columns")
+      linkColumnId <- Future.apply(res.getArray("columns").get[JsonObject](0).getNumber("id"))
+
+      // add rows to tables
+      table1RowId1 <- addRow(1, valuesRow("table1RowId1"))
+      table1RowId2 <- addRow(1, valuesRow("table1RowId2"))
+      table2RowId1 <- addRow(2, valuesRow("table2RowId1"))
+      table2RowId2 <- addRow(2, valuesRow("table2RowId2"))
+    } yield linkColumnId
+  }
+
+  @Test
+  def putLinkValues(): Unit = okTest {
+
+    val putLinks = Json.obj("value" -> Json.obj("from" -> 1, "values" -> Json.arr(1, 2)))
+
+    for {
+    // setup two tables
```

Comment:
Strange intendation? Do we need this comment all the time or can we just rename the function into `setupTwoTables`? :)